paparazzi/src/main/java/app/cash/paparazzi/accessibility/AccessibilityRenderExtension.kt

```diff
@@ -159,6 +159,7 @@ private fun SemanticsNode.accessibilityText(): String? {
   val onClickLabel =
     if (disabled != null) null else config.getOrNull(SemanticsActions.OnClick)?.label?.let { "<on-click>: $it" }
   val heading = if (config.getOrNull(SemanticsProperties.Heading) != null) "<heading>" else null
+  val toggleableState = config.getOrNull(SemanticsProperties.ToggleableState)?.let { "<toggleable: ${it.name}>" }
```

Comment:
Could we do something like this to better reflect how TalkBack handles `ToggleableState`?
```suggestion
  val toggleableState = if (stateDescription != null) {
    // The toggleable state is only read by TalkBack if the state description is not set
    null
  } else {
    config.getOrNull(SemanticsProperties.ToggleableState)?.let {
      when (it) {
        On -> "<checked>"
        Off -> "<not checked>"
        Indeterminate -> "<indeterminate>"
      }
    }
  }
```